### ISSUE TITLE
Can't serve ansible.module_utils.distro._distro using default python3 packages on Rocky Linux.

### ISSUE THREAD

**Issue by sjpb** [mitogen-hq/mitogen#912]
Just trying to get ansible-mitogen working on a Rocky Linux 8.5 system. I'm using:
- python 3.6.8 (which is what `dnf install python3` provides) as I believe that is the latest which has access to the selinux bindings on EL8)
- ansible --version reports `core 2.11.9` (again, the current default pip-installed version for this python).
- I have tried both mitogen 0.3.2 (which is what pip installs) and the master branch.

I have a playbook like this:
```yaml
- hosts: all
  strategy: linear
  gather_facts: no
  become: yes
  tasks:
  - name: Install Python if necessary.
    raw: test -e /usr/bin/python3 || yum install -y python3

- hosts: all
  gather_facts: no
  vars:
    ansible_python_interpreter: /usr/bin/python3
  tasks:
    - shell: hostname
```

it fails on the 2nd play with `The Mitogen master process was unable to serve 'ansible.module_utils.distro._distro'`.

I have searched the issues tracker, but anything relevant looks like it got fixed in 0.3.

Is this an acceptable python / ansible combination? I am a bit confused from the docs to be honest!

Thanks for any help, really interested to try this.

**Comment by sjpb** [comment]
Looks like it does exist for the python I'm using though:
```
(venv) [rocky@testhost mitogen-test]$ python3 -c "from ansible.module_utils import distro; print(distro)"
<module 'distro' from '/home/rocky/mitogen-test/venv/lib64/python3.6/site-packages/distro/__init__.py'>
```

**Comment by sjpb** [comment]
Error part of logs:
```
[DEPRECATION WARNING]: Ansible will require Python 3.8 or newer on the 
controller starting with Ansible 2.12. Current version: 3.6.8 (default, Nov  9 
2021, 14:44:26) [GCC 8.5.0 20210514 (Red Hat 8.5.0-3)]. This feature will be 
removed from ansible-core in version 2.12. Deprecation warnings can be disabled
 by setting deprecation_warnings=False in ansible.cfg.

[WARNING]: 
[mux  3777346] 14:46:38.608236 W mitogen.service: Pool(8fd0, size=32, th='mitogen.Pool.8fd0.0'): call error: Message(0, 1, 0, 110, 1000, b"\x80\x02X'\x00\x00\x00ansible_mitogen.services.ContextServiceq\x00X\x03"..757): builtins.ModuleNotFoundError: The Mitogen master process was unable to serve 'ansible.module_utils.distro._distro'. It may be a native Python extension, or it may be missing entirely. Check the importer debug logs on the master for more information.
  File "<stdin>", line 3669, in _dispatch_one
  File "<stdin>", line 3656, in _parse_request
  File "<stdin>", line 668, in import_module
  File "<stdin>", line 1516, in load_module
  File "master:/home/rocky/slurm-app-mitogen/venv/lib64/python3.6/site-packages/ansible_mitogen/target.py", line 85, in <module>
    import ansible_mitogen.runner
  File "<stdin>", line 1516, in load_module
  File "master:/home/rocky/slurm-app-mitogen/venv/lib64/python3.6/site-packages/ansible_mitogen/runner.py", line 85, in <module>
    import ansible.module_utils.basic
  File "<stdin>", line 1516, in load_module
  File "master:/home/rocky/slurm-app-mitogen/venv/lib64/python3.6/site-packages/ansible/module_utils/basic.py", line 153, in <module>
    from ansible.module_utils.common.sys_info import (
  File "<stdin>", line 1516, in load_module
  File "master:/home/rocky/slurm-app-mitogen/venv/lib64/python3.6/site-packages/ansible/module_utils/common/sys_info.py", line 10, in <module>
    from ansible.module_utils import distro
  File "<stdin>", line 1516, in load_module
  File "master:/home/rocky/slurm-app-mitogen/venv/lib64/python3.6/site-packages/ansible/module_utils/distro/__init__.py", line 45, in <module>
    from ansible.module_utils.distro import _distro as distro
  File "<stdin>", line 1491, in load_module

[WARNING]: 
[task 3777380] 14:46:38.609653 W ansible_mitogen.connection: Connection failed; stack configuration was:
({'kwargs': {'check_host_keys': 'ignore',
             'compression': True,
             'connect_timeout': 10,
             'hostname': '10.60.102.213',
             'identities_only': False,
             'identity_file': '/home/rocky/.ssh/id_rsa_slurm_app_ci',
             'keepalive_count': 10,
             'keepalive_interval': 30,
             'password': None,
             'port': None,
             'python_path': ['/usr/bin/python3'],
             'remote_name': None,
             'ssh_args': ['-o',
                          'ControlMaster=auto',
                          '-o',
                          'ControlPersist=240s',
                          '-o',
                          'PreferredAuthentications=publickey',
                          '-o',
                          'UserKnownHostsFile=/dev/null'],
             'ssh_debug_level': None,
             'ssh_path': 'ssh',
             'username': 'rocky'},
  'method': 'ssh'},)
```

**Comment by sjpb** [comment]
```
(venv) [rocky@testhost slurm-app-mitogen]$ python3
Python 3.6.8 (default, Nov  9 2021, 14:44:26) 
[GCC 8.5.0 20210514 (Red Hat 8.5.0-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from ansible.module_utils import distro
>>> from ansible.module_utils.distro import _distro as distro
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name '_distro'
>>> 
```

**Comment by sjpb** [comment]
On the other hand, after looking at `venv/lib64/python3.6/site-packages/distro/distro.py`, this works:
```python
>>> from ansible.module_utils.distro.distro import _distro as distro
```

**Comment by sjpb** [comment]
Agh. This fixes it:
```shell
 pip install distro==1.6.0
```.

Looks like this is https://github.com/mitogen-hq/mitogen/issues/906

**Comment by moreati** [comment]
This should now be fixed in master along with #906, by #913
